include/nbl/ui/CWindowManagerXCB.h

```diff
@@ -0,0 +1,94 @@
+#ifndef C_WINDOW_MANAGER_XCB
+#define C_WINDOW_MANAGER_XCB
```

Comment:
you want to start and end your headerguard with `_` and include full namespace and the `INCLUDED` suffix